lib/rjgit_adapter/git_layer_rjgit.rb

```diff
@@ -190,31 +190,36 @@ def checkout(path, ref, options = {})
       
       # rev_list({:max_count=>1}, ref)
       def rev_list(options, *refs)
-        raise "Not implemented"
+        raise 'Not implemented'
       end
       
       def ls_files(query, options = {})
         ref = Gollum::Git.canonicalize(options[:ref])
-        result = RJGit::Porcelain.ls_tree(@git.jrepo, options[:path], ref, {:recursive => true}).select {|object| object[:type] == "blob" && !!(::File.basename(object[:path]) =~ /#{query}/i) }
+        result = RJGit::Porcelain.ls_tree(@git.jrepo, options[:path], ref, {:recursive => true}).select {|object| object[:type] == 'blob' && !!(::File.basename(object[:path]) =~ /#{query}/i) }
         result.map do |r|
           r[:path]
         end
       end
-
-      def revert_path(path, sha1, sha2, ref = 'HEAD^{tree}')
+      
+      def revert_path(path, sha1, sha2, ref = 'HEAD^{commit}')
+        result, paths = revert(path, sha1, sha2, ref)
```

Comment:
Maybe too late, but I would prefer `_paths` to `paths` to indicate explicitly that the variable will not be used.

Moreover, perhaps `revert_by_path` would be more suitable, because on a slightly different reading of `revert_path` it could be a method that returns a path after a revert or something to that effect, and then it's strange to see `paths` being ignored.